go/cmd/server/server_test.go

```diff
@@ -58,21 +66,51 @@ func getClient(ctx context.Context) (serving.ServingServiceClient, func()) {
 }
 
 func TestGetFeastServingInfo(t *testing.T) {
-	t.Skip("@todo(achals): feature_repo isn't checked in yet")
 	ctx := context.Background()
-	client, closer := getClient(ctx)
+	// Pregenerated using `feast init`.
+	client, closer := getClient(ctx, "../../internal/test")
 	defer closer()
 	response, err := client.GetFeastServingInfo(ctx, &serving.GetFeastServingInfoRequest{})
 	assert.Nil(t, err)
 	assert.Equal(t, feastServerVersion, response.Version)
 }
 
-func TestGetOnlineFeatures(t *testing.T) {
-	t.Skip("@todo(achals): feature_repo isn't checked in yet")
+func TestGetOnlineFeaturesSqlite(t *testing.T) {
 	ctx := context.Background()
-	client, closer := getClient(ctx)
+	// Pregenerated using `feast init`.
+	client, closer := getClient(ctx, "../../internal/test")
 	defer closer()
-	response, err := client.GetOnlineFeatures(ctx, &serving.GetOnlineFeaturesRequest{})
+	entities := make(map[string]*types.RepeatedValue)
+	entities["driver_id"] = &types.RepeatedValue{
+		Val: []*types.Value{
+			{Val: &types.Value_Int64Val{Int64Val: 1001}},
+			{Val: &types.Value_Int64Val{Int64Val: 1003}},
+			{Val: &types.Value_Int64Val{Int64Val: 1005}},
+		},
+	}
+	request := &serving.GetOnlineFeaturesRequest{
+		Kind: &serving.GetOnlineFeaturesRequest_Features{
+			Features: &serving.FeatureList{
+				Val: []string{"driver_hourly_stats:conv_rate", "driver_hourly_stats:acc_rate", "driver_hourly_stats:avg_daily_trips"},
+			},
+		},
+		Entities: entities,
+	}
+	response, err := client.GetOnlineFeatures(ctx, request)
+	expectedEntityValuesResp := []*types.Value{
+		{Val: &types.Value_Int64Val{Int64Val: 1001}},
+		{Val: &types.Value_Int64Val{Int64Val: 1003}},
+		{Val: &types.Value_Int64Val{Int64Val: 1005}},
+	}
+	expectedFeatureNamesResp := []string{"driver_id", "conv_rate", "acc_rate", "avg_daily_trips"}
 	assert.Nil(t, err)
 	assert.NotNil(t, response)
+
+	// Columnar so first column should have column names of all features
+	assert.True(t, reflect.DeepEqual(response.Results[0].Values, expectedEntityValuesResp))
```

Comment:
let's add here check for feature values as well, since we read them from the source file

Reply:
DOne.